source/MaterialXRuntime/RtStage.cpp

```diff
@@ -11,6 +11,8 @@
 
 #include <MaterialXRuntime/Private/PvtStage.h>
 
+#include <MaterialXCore/Element.h>
```

Comment:
Is this include needed?

Reply:
It was needed for the UI folder constant, but now that that has been removed, I'll remove this.